theatres/korsac-2023/TT30_KorsacTemplate.VTF_Korsac/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,27 @@
+After the latest elections in Korsac the losing side decided to take up arms in an attempt to topple the winning government and take power for themselves.
+Since the local defense forces are having trouble suppressing this rebellion Synixe has been hired to perform COIN (Counter Insurgency) operations in the region in order to aid the winning government.
```

Comment:
```suggestion
Since the local defense forces are having trouble suppressing this rebellion, Synixe has been hired to perform COIN (Counter Insurgency) operations in the region in order to aid the winning government.
```